cmd/tsbs_generate_data/main.go

```diff
@@ -275,10 +284,13 @@ func getSerializer(sim common.Simulator, format string, out *bufio.Writer) seria
 	switch format {
 	case formatCassandra:
 		return &serialize.CassandraSerializer{}
+
```

Comment:
prefer no spaces here

Reply:
Ok, let it be no spaces, however it seems to be excessively dense (IMHO)

Reply:
Updated.